pulp_ansible/app/tasks/collections.py

```diff
@@ -220,7 +220,7 @@ def _update_highest_version(collection_version):
     if not last_highest:
         collection_version.is_highest = True
         return None
-    if semver.compare(collection_version.version, last_highest.version) > 0:
+    if Version(collection_version.version) > Version(last_highest.version):
```

Comment:
```python
In [1]: import semantic_version as semver

In [2]: semver.compare("9.1.0-dev.1+40688.git966d0b3", "9.1.0-dev.1+40390.gita455009")
Out[2]: NotImplemented

In [3]: semver.Version("9.1.0-dev.1+40688.git966d0b3") > semver.Version("9.1.0-dev.1+40390.gita455009")
Out[3]: False

In [4]: semver.Version("9.1.0-dev.1+40688.git966d0b3") < semver.Version("9.1.0-dev.1+40390.gita455009")
Out[4]: False

In [5]: semver.Version("9.1.0-dev.1+40688.git966d0b3") == semver.Version("9.1.0-dev.1+40390.gita455009")
Out[5]: False
```

Reply:
This is looking good to me. Now we're asking it a question it can answer!